test/Lava.spec.ts

```diff
@@ -14,15 +18,20 @@ describe('Lava', () => {
     });
 
     beforeEach(function () {
-        scene.onTestStart(() => {});
+        scene.onTestStart((_lava, _player) => {
+            lava = _lava;
+            playerMock = _player;
+        });
         phaserTester.onTestStart(this.currentTest.title);
     });
 
-    it('should move up', () => {
-        assert.fail('Not implemented');
+    it('should move up', async () => {
+        const oldLavaY: number = lava.y;
```

Comment:
Can't see the lava when running tests on browser (but it still appears functionally correct)

- [ ] Assess why Lava rectangle doesn't appear to be visible in test scene and fix.